docs/environments.md

```diff
@@ -0,0 +1,40 @@
+# Environments
+
+An Environment describes a configuration tailored for a context of any kind
+whatsoever.  
+Such a context might be `dev` and `prod` environments, blue/green deployments or the
+same application available in multiple regional zones / datacenters.
+
+An environment does not need to be created, it rather just exists as soon as a
+main.jsonnet is found somewhere in the tree of a `rootDir`. This also means that
+an environment by definition is equivalent to a [`baseDir`](directory-structure.md#base-directory-basedir).
+
+## Configuration
+To correctly deal with an environment, Tanka needs some additional information
+about it.
+
```

Comment:
Presumably this additional info is expected in `spec.json` next to `main.jsonnet`? It doesn't says so.

Reply:
Oh yeah, missed to add that info